backend/compact-connect/lambdas/purchases/data_model/schema/compact.py

```diff
@@ -9,8 +11,16 @@
 COMPACT_TYPE = 'compact'
 
 
+class CompactFeeType(Enum):
+    FLAT_RATE = 'FLAT_RATE'
+
+    @staticmethod
+    def from_str(label: str) -> 'CompactFeeType':
```

Comment:
This is the pattern I'm used to, as sometimes you end up with a unique mapping between an API and the underlying values you set so this gives you flexibility with those scenarios. Let me know if you would like to use the built in syntax.

Reply:
I'm good either way, if we anticipate using Enums in many places, which we hopefully will for groups of static string values, then it could be useful to enforce that pattern. I leave it up to your call. 😄

Reply:
Ok, let's go for it, then.

Reply:
I've added a common CCEnum type in my latest commit.